framework/PostProcessors/BasicStatistics.py

```diff
@@ -1085,16 +1085,18 @@ def run(self, inputIn):
       @ In,  inputIn, object, object contained the data to process. (inputToInternal output)
       @ Out, outputDict, dict, Dictionary containing the results
     """
-    input = self.inputToInternal(inputIn)
+    inputAdapted = self.inputToInternal(inputIn)
```

Comment:
Good catch. We should make sure this doesn't occur anywhere else in the code since this was overwriting the built-in [input](https://docs.python.org/3/library/functions.html#input) function.